### PR TITLE
aliging to docker master branch and new docker compose

### PR DESCRIPTION
**Pull request by jeanmi151**
connected with https://github.com/georchestra/docker/pull/187

need to realign https://github.com/georchestra/helm-georchestra too

### CODE REVIEW

**Comment by edevosc2c** [comment]
note: we will need to align all the existing datadir from all the clients too

**Review by edevosc2c — APPROVED** [review]
good but will need to validate that it works with kubernetes

File: gateway/roles-mappings.yaml [hunk 1]
```diff
@@ -29,4 +29,20 @@
 #        - ROLE_USER
 #      '[ROLE_GP.GDI.ADMINISTRATOR]':
 #        - ROLE_ADMINISTRATOR
-
+ 
+georchestra:
+  gateway:
+    roles-mappings:
+      '[ROLE_GP.GDI.*]':
```
Comment (fvanderbiest) [inline gateway/roles-mappings.yaml:36]: cc @pmauduit this reads like DT specifics ...

**Review by fvanderbiest — APPROVED** [review]

File: gateway/roles-mappings.yaml (hunk 1 above)
Comment (pmauduit) [reply gateway/roles-mappings.yaml:36]: indeed, this is a feature needed by DT, but we can probably leave it commented here

**Pull request merged**